steps/terraform-precheck.yaml

```diff
@@ -0,0 +1,65 @@
+parameters:
+  - name: adolibrary
+    default: cnp-azuredevops-libraries
+
+  - name: environment
+    default: sandbox
```

Comment:
why is there a default here?

Reply:
Not required. Left  over from debugging and now removed.